trusts/decorators.py

```diff
@@ -42,6 +108,7 @@ def _wrapped_view(request, *args, **kwargs):
         return _wrapped_view
     return decorator
 
+
```

Comment:
Well, PEP-8 doesn't agree with that:

> Surround top-level function and class definitions with two blank lines.
> Method definitions inside a class are surrounded by a single blank line.

https://www.python.org/dev/peps/pep-0008/#blank-lines
But I can delete these blank lines if you want.